docs/sql-ref-ansi-compliance.md

```diff
@@ -454,6 +454,7 @@ Below is a list of all the keywords in Spark SQL.
 |CONCATENATE|non-reserved|non-reserved|non-reserved|
```

Comment:
some of the keywords are missing here. also, check [parser PR](https://github.com/apache/spark/pull/46665) to see all places where I had to register/test/etc the addition of `BEGIN` keyword, you'll probably need to do the same...

Reply:
Done.